features/pages/projects_page.rb

```diff
@@ -0,0 +1,15 @@
+class ProjectsPage < HomePage
+
+  include PageObject
+
+  page_url DEFAULT_PAGE + '/projects'
+
+  ### Elements
+
+  link(:new_project, class: 'icon icon-add')
+  links(:projects_titles, class: 'project root leaf')
```

Comment:
we can use one class name here 
replace locator with one class where it's possible